CMake build failed of Prometheus exporter test fails

Got below error when building prometheus exporter test.
```console
collector_test.cc.obj : error LNK2019: unresolved external symbol "
public:
    static class opentelemetry::v1::sdk::resource::Resource __cdecl opentelemetry::v1::sdk::resource::Resource::Create(
        class opentelemetry::v1::sdk::common::AttributeMap const &,
        class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)
referenced in function 
struct opentelemetry::v1::sdk::metrics::ResourceMetrics __cdecl `anonymous namespace'::CreateDropPointData(void)
exporters\prometheus\test\collector_test.exe : fatal error LNK1120: 1 unresolved external
```

--- a/exporters/prometheus/test/CMakeLists.txt
+++ b/exporters/prometheus/test/CMakeLists.txt
@@ -15,7 +15,7 @@ else()
     add_executable(${testname} "${testname}.cc")
     target_link_libraries(
       ${testname} ${GTEST_BOTH_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT}
-      prometheus_exporter prometheus-cpp::pull)
+      prometheus_exporter prometheus-cpp::pull opentelemetry_resources)
     gtest_add_tests(
       TARGET ${testname}
       TEST_PREFIX exporter.

